### ISSUE TITLE
Suggestion: Add rule(s) to disallow debugging-related Cypress calls

### ISSUE THREAD

**Issue by cjcrandall** [cypress-io/eslint-plugin-cypress#74]
#57 already covers `.only`, but I think it would be valuable to either have a single rule, or individual rules, to cover other Cypress calls that are usually just for debugging and we don't want to accidentally commit to source control, including:

* `cy.pause`
* `xdescribe/xit/.skip`
* `.only` (Again, see #57)
* `cy.debug`

**Comment by distante** [comment]
I am really interested in this also! I see my self commiting `.debug()` and `.pause()`  calls all the time :(

**Comment by jcupps** [comment]
Hi, it looks like chained `.pause()` calls are not detected by this rule, which is confirmed in the [docs](https://github.com/cypress-io/eslint-plugin-cypress/blob/master/docs/rules/no-pause.md):
> Valid:
>
> ```js
> // only the parent cy.pause command is detected
> cy.get('selector').pause();
> ```

Is there any particular reason for that, or has it just not been implemented yet? Thanks

**Comment by MikeMcC399** [comment]
- This is partially implemented in PR #216
